sqlitecluster/SQLiteNode.cpp

```diff
@@ -1416,98 +1447,10 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
             throw e;
         }
     } else if (SIEquals(message.methodLine, "BEGIN_TRANSACTION")) {
-        // BEGIN_TRANSACTION: Sent by the MASTER to all subscribed slaves to begin a new distributed transaction. Each
-        // slave begins a local transaction with this query and responds APPROVE_TRANSACTION. If the slave cannot start
-        // the transaction for any reason, it is broken somehow -- disconnect from the master.
-        // **FIXME**: What happens if MASTER steps down before sending BEGIN?
-        // **FIXME**: What happens if MASTER steps down or disconnects after BEGIN?
-        bool success = true;
-        uint64_t masterSentTimestamp = message.calcU64("masterSendTime");
-        uint64_t slaveDequeueTimestamp = STimeNow();
-        if (!message.isSet("ID")) {
-            STHROW("missing ID");
-        }
-        if (!message.isSet("NewCount")) {
```

Comment:
did we lose some of these protections in the new version? I was able to suss out some of them in the new code, but not all of them.